crypto/ec/ec_backend.c

```diff
@@ -150,20 +150,114 @@ char *ossl_ec_pt_format_id2name(int id)
     return NULL;
 }
 
-int ossl_ec_group_todata(const EC_GROUP *group, OSSL_PARAM_BLD *tmpl,
-                         OSSL_PARAM params[], OSSL_LIB_CTX *libctx,
-                         const char *propq,
-                         BN_CTX *bnctx, unsigned char **genbuf)
+static int ec_group_explicit_todata(const EC_GROUP *group, OSSL_PARAM_BLD *tmpl,
```

Comment:
This function is used by get_params() AND export(). 
Should we be doing the explicit stuff during export or not?
The other case related to this is  fromdata -> merge params -> todata..
Does anyone have any thoughts on this?

Reply:
I do not see a problem with this apart from a little inefficiency as on import we first check whether there is a group name and if it is there the explicit curve parameters are ignored.

Reply:
I do not know if there would be some nice way how to get around this inefficiency.

Reply:
Yeah it does seem quite inefficient. Using OSSL_PARAM_locate() to check if we want that specific parameter before extracting/processing it might be better than waiting for the logic inside ossl_param_build_set_XXX() to do this.

Reply:
Do the locate, they are fast enough (and can be made faster if they are ever a bottleneck).